index.html

```diff
@@ -340,30 +340,11 @@
         }
 
         .sp-tab-page {
-            visibility: hidden;
-            /* display: none; */
-            flex: 1 1 auto;
-            /* overflow: scroll; */
-            overflow-y: scroll;
-            padding: 12px 0;
+            display: none;
+            padding: 10px 0;
```

Comment:
```suggestion
            padding: 10px 0;
```
Can you change this to 7px padding to all sides?

Reply:
Done